src/core/utils/index.js

```diff
@@ -376,3 +376,26 @@ export const findFileForPlatform = ({ userAgentInfo, platformFiles }) => {
   const platform = agentOsName && userAgentOSToPlatform[agentOsName];
   return (platform && platformFiles[platform]) || platformFiles[OS_ALL];
 };
+
+export const getLocalizedTextWithLinkParts = ({
```

Comment:
I don't really like adding code that isn't covered by Flow, but this file is not covered by Flow and it's quite large, so I guess you don't want to tackle converting the whole thing to Flow at this point. You could maybe create a separate file in `src/core/utils/` called `i18n.js` as this utility function is specific to `i18n`. Then you could make sure that file is covered by Flow.

What do you think?

Reply:
I like this idea :)